Tests/AblyChatTests/DefaultInternalLoggerTests.swift

```diff
@@ -0,0 +1,49 @@
+@testable import AblyChat
+import XCTest
+
+class DefaultInternalLoggerTests: XCTestCase {
+    func test_defaults() {
+        let logger = DefaultInternalLogger(logHandler: nil, logLevel: nil)
+
+        XCTAssertTrue(logger.logHandler is DefaultLogHandler)
+        XCTAssertEqual(logger.logLevel, .info)
```

Comment:
Shouldn't only errors be logged by default?

Reply:
You're right, that's what JS does. Will change.

Reply:
Done